doc/stages/filters.voxeldownsize.rst

```diff
@@ -39,9 +40,11 @@ cell
 
 mode
   Mode for voxel based filtering. [Default: voxelcenter]
-  **voxelcenter**: Retain center of a populated voxel. Coordinates of a first point detected will be modified and set to the center of a populated voxel.
-  **firstinvoxel**: Retain first point detected in each voxel.
+  **voxelcenter**: Coordinates of the first point found in each voxel will
+  be modified to be the center of the voxel.
+  **firstinvoxel**: Only the first point found in each voxel is retained.
 
 .. warning::
-    If you choose **voxelcenter** mode, you are loosing your dimensional data/point coordinates.
-  
\ No newline at end of file
+    If you choose **voxelcenter** mode, you are overwriting the X, Y and Z
+    values of retained points.
```

Comment:
Originally he says "losing your dimensional data", which I suppose isn't true, the dimension still exists, but it is invalidated in the sense that it is simply inheriting the dimension data of the first point, which may or may not be a good thing. Maybe we should still try to capture this fact in some manner.

Reply:
Updated.  Let me know if you have better wording.